pyfolio/tears.py

```diff
@@ -64,8 +64,14 @@ def create_returns_tear_sheet(returns, live_start_date=None,
         Daily non-cumulative returns of the first benchmark.
     return_fig : boolean, optional
         If True, returns the figure that was plotted on.
+    set_context : boolean, optional
+        If True, set default plotting style context.
     """
 
+    if set_context:
```

Comment:
@richafrank I think ExitStack requires contextlib2 as a dependency on Py2?  We may already have that though given that we've got a lot of heavyweight SciPy modules as dependencies.